### PR TITLE
Fix duplicates in WinterthurCHSpider

### LINKED ISSUE #9099: Duplicate id in winterthur_ch
The id `tWidPqOkZrC2-MsLg8PoKAHXqvY=` is assigned twice in spider `winterthur_ch`. I'm not sure if this is the only case of duplicate id but it may be worth checking these in an automated way before publishing the data.

### PR DESCRIPTION
**Pull request by CloCkWeRX**
Fix #9099 ?

### CODE REVIEW

File: locations/spiders/winterthur_ch.py [hunk 1]
```diff
@@ -70,6 +72,10 @@ def parse_playgrounds(self, response):
     def parse_benches(self, response):
         for m in response.json()["FeatureCollection"]["member"]:
             if f := m.get("SitzbankWfs"):
+                ref = f["Banknummer"]
+                if ref in self.ids_seen:
```
Comment (CloCkWeRX) [inline locations/spiders/winterthur_ch.py:76]:
```suggestion
                if ref in self.seen_ids:
```

File: locations/spiders/winterthur_ch.py [hunk 2]
```diff
@@ -88,8 +94,9 @@ def parse_benches(self, response):
                         "material": self.parse_bench_material(btype),
                         "material:wikidata": self.parse_bench_material_wikidata(btype),
                     },
-                    "ref": f["Banknummer"],
+                    "ref": ref,
                 }
+                self.seen_ids.append(ref)
```
Comment (CloCkWeRX) [inline locations/spiders/winterthur_ch.py:99]:
```suggestion
                self.seen_ids.add(ref)
```

**Comment by CloCkWeRX** [comment]
Maybe not.

**Comment by vfosnar** [comment]
I don't think dropping on id collision is the right thing to do. Implementing id generation based on both ref and position would be a better way to handle this.

**Comment by CloCkWeRX** [comment]
Its the exact same bench twice in the dataset.

**Comment by iandees** [comment]
The dupe detection pipeline step should be doing exactly this. We should figure out why that isn't dropping it as expected.

**Comment by CloCkWeRX** [comment]
It does attempt to run, the problem with this dataset is it's a composite.
The benches have refs.
Others don't.
So, this change targets just the benches component and adds basically the duplication behavior, or is meant to.
Potentially the spider could be split in two.

**Comment by iandees** [comment]
Yea, I would rather see this be two separate spiders than to duplicate the deduplication process.

**Comment by CloCkWeRX** [comment]
Effort kind of high for... one ... doubled bench entry

**Pull request closed without merging**